kong/clustering/rpc/manager.lua

```diff
@@ -270,6 +270,16 @@ function _M:_meta_call(c, meta_cap, node_id)
     list = capabilities_list,
   }
 
+  -- tell outside that rpc is ready
+  local worker_events = assert(kong.worker_events)
+
+  -- notify all workers (worker 0 or privileged agent)
+  local ok, err = worker_events.post("clustering:jsonrpc", "connected",
```

Comment:
This doesn't feels right, RPC connection is per worker, worker events will flood the event into all workers (even not yet connected worker). I think we should use a simple callback mechanism that is within the worker instead.

Reply:
OK, change to `post_local()`

Reply:
https://github.com/Kong/kong/pull/14034/commits/76522288d80115cb886a0edb89c0ef1568e26840 fixed